src/utils/convert.js

```diff
@@ -495,15 +559,13 @@ const renderIndex = (legacyApp) => {
     AFTER_RESPONSES: getAfterResponses(legacyApp),
   };
 
-  return renderAuth(legacyApp)
-    .then((auth) => {
-      templateContext.AUTHENTICATION = auth;
-      return getHeader(legacyApp);
-    })
+  return getHeader(legacyApp)
     .then((header) => {
       templateContext.HEADER = header;
 
-      const importLines = [];
+      const importLines = [
+        "const authentication = require('./authentication');"
```

Comment:
Like I mentioned above, we should only do this if the app has auth.

Reply:
Fixed in f837e3a.